src/modules/core/components/Heading/Heading.css

```diff
@@ -78,6 +78,10 @@
   font-size: var(--size-small);
 }
 
+.sizeSmallish {
```

Comment:
I agree with Arren, I would not add another size to the heading to be consistent with the rest of the dapp.